docs/web/examples/index.rst

```diff
@@ -9,8 +9,9 @@ Examples
 twisted.web.client
 ------------------
 
-- :download:`getpage.py` - use ``twisted.web.client.getPage`` to download a web page.
-- :download:`dlpage.py` - add callbacks to ``twisted.web.client.downloadPage`` to display errors that occur when downloading a web page
+- :download:`httpclient.py` - use ``twisted.web.client.Agent`` to download a web page.
+- (deprecated) :download:`getpage.py` - use ``twisted.web.client.getPage`` to download a web page.
+- (deprecated) :download:`dlpage.py` - add callbacks to ``twisted.web.client.downloadPage`` to display errors that occur when downloading a web page
```

Comment:
I'll note that when I deprecated `method`, I also removed the documentation for it from our narrative docs. I guess it depends on how bad it is if someone uses it. (I personally am on the side of deleting it, and having the documentation for the One Way. Previous versions of the docs still exist on our website forever.).